contracts/LockedTokens.cdc

```diff
@@ -99,11 +99,11 @@ pub contract LockedTokens {
 
         /// Optional NodeStaker resource. Will only be filled if the user
         /// signs up to be a node operator
-        pub var nodeStaker: @FlowIDTableStaking.NodeStaker?
+        pub(set) var nodeStaker: @FlowIDTableStaking.NodeStaker?
 
         /// Optional NodeDelegator resource. Will only be filled if the user
         /// signs up to be a delegator
-        pub var nodeDelegator: @FlowIDTableStaking.NodeDelegator?
+        pub(set) var nodeDelegator: @FlowIDTableStaking.NodeDelegator?
```

Comment:
Why make these fields publicly writeable? Isn't that a security problem?

Reply:
This object is only in the locked account, so it can only be accessed via a transaction co-signed by the user and the token admin. We need it right now to fix some accounts that were compromised by an operation we performed recently

Reply:
Sounds good, thank you for the explanation and context 👍